src/enzyme_ad/jax/Passes/EnzymeHLOOpt.cpp

```diff
@@ -8922,8 +8989,9 @@ struct EnzymeHLOOptPass
       patterns.add<FullReduceReshapeOrTranspose>(context);
 
     if (passses & 1)
-      patterns.add<SliceTranspose, SliceReshapeTranspose, SliceBroadcast>(
-          context);
+      patterns.add<
+          // SliceTranspose,
+          TransposeSlice, SliceReshapeTranspose, SliceBroadcast>(context);
```

Comment:
we should make this a separate pass flag from SliceTranspose and SliceReshapeTranspose [both of which might undo SliceTranspose]. Aka not passses & 1, but passses & [another number]]